### ISSUE TITLE
Blurred Title on non-journal pages

### ISSUE THREAD

**Issue by dandusty** [solariz/logseq-neon-theme#7]
I'm using v0.4 and it looks great, but when I visit a page other than journal pages, the heading is blurred.  I don't have anything in custom.css.

**Comment by dandusty** [comment]
<img width="732" alt="image" src="https://user-images.githubusercontent.com/74432202/208929398-613ca83b-b38d-4164-b836-35887b84e978.png">

**Comment by solariz** [comment]
Yep, can confirm had that by myself after the recent logseq update.
I pushed a 0.5 Update just now, it take some hours until the update is available in the marketplace, give it a try.
